central/resourcecollection/service/service_impl_test.go

```diff
@@ -51,7 +51,7 @@ func (suite *CollectionServiceTestSuite) TearDownSuite() {
 }
 
 func (suite *CollectionServiceTestSuite) TestListCollectionSelectors() {
-	if !features.ObjectCollections.Enabled() {
+	if !env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
Remove skip from here and skip in `SetupSuite` right after `suite.T().Setenv(env.PostgresDatastoreEnabled.EnvVar(), "true")`

Reply:
+1 to Charmik's suggestion